screens/Activate.js

```diff
@@ -38,7 +38,12 @@ const ActivateScreen = () => {
             WebBrowser.openBrowserAsync(
               'https://docs.google.com/forms/d/e/1FAIpQLSdZ1EBI_kCqt8xtK1n1PBfcBUlHFPl45o-9Ls3O2srwejpjGw/viewform?vc=0&c=0&w=1',
               { toolbarColor: Colors.darkestGray }
-            );
+            ).catch(err => {
```

Comment:
handle [this issue](https://github.com/expo/expo/issues/6918)
i'm assuming it'll be fixed in the next expo release